data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1228 @@
+
+			branch "boarded epidote"
+				has "scin: entered epidote"
+
+			branch "saw epidote"
+				has "scin: approached epidote"
+
+			`	Once you get close enough, the oblong object reveals itself to be some sort of large Gegno ship with the coloring of the Scin faction. What it's doing this far away from their home space is a mystery, much like the moon it's stationed on. After a few seconds, it launches into the atmosphere and leaves the moon. There doesn't seem to be anything left where it was parked, nor any trace of what the Scin were doing here to begin with.`
+				decline
+
+			label "saw epidote"
+			`	Once you get close enough, the oblong object is revealed to be a large Gegno Scin ship, similar to the one you encountered during an interaction with the Scin a while ago. What it's doing this far away from their home space is a mystery, much like the moon it's stationed on. Or at least, was stationed on, as after a few seconds it launches into the atmosphere and leaves the moon. Perhaps it was some sort of surveillance ship spying on the nearby Gegno Vi systems.`
+				decline
+
+			label "boarded epidote"
+			`	Once you get close enough, the oblong object reveals itself to be one of the Gegno Scin's large research ships, the same kind that you entered during an interaction with the Scin a while ago. It's possible the Scin found the mysteriousness of Cyife worth traveling so far from their home space to investigate. You'll never know for certain as after a few seconds, the ship launches into the atmosphere and disappears into space. They also didn't seem to leave anything behind, though the site the ship was parked at does appear to be one of the many strange regions of the moon.`
+				decline
+
+
+mission "Battle over Zydee"
+	invisible
+	landing
+	to offer
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	to complete
+		never
+	source "Zydee"
+	on offer
+		"gegno: encounters" ++
+		conversation
+			`Although this planet is uninhabited, there are more than a few Gegno Scin ships currently parked across the surface. It seems like there's an ongoing operation to continue constructing what infrastructure has been built in the past, and the hard-working Scin pay little to no attention to your visit.`
+			`	However, a few moments later, the Scin suddenly start scurrying to their ships. You haven't gotten close enough to alarm them, at least from your experiences with them so far. They begin taking off one by one in a hurried fashion, but not towards you - the <ship>'s sensors suddenly begin picking up a large Gegno Vi fleet entering the system. It appears that this place is turning into a battlefield, and getting caught up in the middle of it is likely something you should avoid.`
```

Comment:
```suggestion
			`	However, a few moments later, the Scin suddenly start scurrying to their ships. Based on your experiences so far, you haven't gotten close enough to alarm them. They begin taking off one by one in a hurried fashion, but not toward you - the <ship>'s sensors suddenly begin picking up a large Gegno Vi fleet entering the system. It appears that this place is turning into a battlefield, and getting caught up in the middle of it is likely something you should avoid.`
```
Just trying to avoid using `them` twice in the second sentence, sounds a little funky to me. Open to suggestions!